libqtile/group.py

```diff
@@ -267,6 +271,8 @@ def add(self, win, focus=True, force=False):
                 i.add_client(win)
         if focus:
             self.focus(win, warp=True, force=force)
+        else:
+            self.layout_all(focus=False)
```

Comment:
Hmm. Is this a bug fix that is unrelated to this PR? We always want to layout when a window is added, right?

Reply:
Yes. It's not really unrelated though as probably you don't want new windows to float during the prompt being active.

Reply:
Right, I agree we want this patch in the case you're trying to fix in this PR. I'm suggesting we probably want it in other cases too, and that perhaps it could/should be its own separate patch?